test/k8sT/Services.go

```diff
@@ -1979,6 +1982,10 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 			}
 		})
 
+		SkipItIf(helpers.DoesNotRunOn419OrLaterKernel, "Tests ClusterIP with sessionAffinity from pods", func() {
+			testSessionAffinity(false, true)
```

Comment:
How is that different from the existing `Tests NodePort with sessionAffinity`? Maybe we need to redeploy Cilium with KPR disabled here?

Reply:
Extended the skip condition to make it more obvious that it should run with KPR=disabled.

Reply:
But when is that condition ever true? `helpers.RunsOn419OrLaterKernel() && helpers.RunsWithoutKubeProxyReplacement()`

Reply:
On GKE it should be true.

Reply:
It isn't: https://github.com/cilium/cilium/blob/45689ece6f242c4c2546a3f32ddd80f6464f6e24/test/helpers/utils.go#L567-L572